### ISSUE TITLE
Manually passed parameters are ignored when compiling

### ISSUE THREAD

**Issue by GustavBjordal** [MiniZinc/MiniZincIDE#121]
When compiling (not running) a model that requires some parameters but has no dzn file, the IDE seems to not properly pass the manually defined parameters to the compiler.

```
int: n;
var int: x;
```
Doing `cmd + b` or `run>compile` or `MiniZinc>compile` in the IDE and providing for example the value `42` in the prompt gives the following output:

```
Compiling untitled_model.mzn, additional arguments -D n=42;
MiniZinc: type error:   symbol error: variable `n' must be defined (did you forget to specify a data file?)
/path/to/untitled_model.mzn:1.1-6

Process finished with non-zero exit code 1
Finished in 51msec
```

Tested on version 2.5.1 on macOS.

**Comment by zayenz** [comment]
Seems to be a similar case to #118, which is fixed in the develop branch.

**Comment by cyderize** [comment]
I believe this should now be fixed in 2.5.2.
